tembo-cli/src/cmd/logs.rs

```diff
@@ -0,0 +1,108 @@
+use crate::apply::{get_instance_id, get_instance_settings};
+use crate::cli::context::{get_current_context, Environment, Profile};
+use anyhow::Result;
+use clap::Args;
+use reqwest::blocking::Client;
+use reqwest::header::{HeaderMap, HeaderValue, AUTHORIZATION};
+use serde::{Deserialize, Serialize};
+use temboclient::apis::configuration::Configuration;
+
+#[derive(Args)]
+pub struct LogsCommand {
+    #[clap(short, long)]
+    pub verbose: bool,
```

Comment:
I think logs command should not have verbose as a field since that's already a global option, but I may be mistaken about that.

Reply:
You're right. It shouldn't have it.